en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md

```diff
@@ -99,52 +111,46 @@ Follow the steps given below to create the data service file:
     | URL                                | jdbc:mysql://localhost:3306/Employees |
     | User Name                          | root                                  |
 
-5.  Save the data service.
+4.  Click **Test Connection** to expand the section.
+    ![](../../assets/img/tutorials/data_services/test_connection.png)
```

Comment:
@DMadhuranga - Shall we add a screenshot for this? Same as for 'datasource connection details'.
Let's keep this pattern of screenshots consistent.